.github/dependabot.yaml

```
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
```

```suggestion
      interval: "monthly"
```
recently I saw a way to group all into one PR so it would limit nb open PRs
https://github.com/Borda/pytest-rerunfailures/blob/3b9ad82537002dd3d61ae3e980a049883db91c36/.github/dependabot.yml#L13-L18